arelle/plugin/validate/ESEF/ESEF_Current/Image.py

```diff
@@ -182,18 +192,19 @@ def checkSVGContentElt(
             if elt.tag != "{http://www.w3.org/2000/svg}svg":
                 modelXbrl.error(f"{guidance}.imageFileCannotBeLoaded",
                                 _("Image SVG has root element which is not svg"),
-                                modelObject=imgElt)
+                                modelObject=imgElts)
             rootElement = False
         eltTag = elt.tag.rpartition("}")[2] # strip namespace
         if eltTag == "image":
-            validateImage(baseUrl, getHref(elt), modelXbrl, val, elt, "", guidance)
+            imgElts.insert(0, elt)
```

Comment:
Does `.insert(0` make the "deepest" image be `imgElts[0]`?  It's subjective, but it seems more logical to me to list the references in outer->inner order so that the iXBRL `<img>` element is `imgElts[0]`.

Does this `.insert` get "undone" somewhere that I'm missing?  If ixbrl->svg->[sub0,sub1], it looks like imgElts would contain [sub1,sub0,svg,ixbrl], which seems wrong?

Reply:
Good catch on sub0, sub1. inner > outer made more sense to me (prioritize the element that actually has the error), but I don't feel strongly. I'll flip it.